ansible/roles/cli/tasks/deploy.yml

```diff
@@ -3,26 +3,19 @@
 
 - name: "ensure nginx directory for cli exists"
   file:
-    path: "{{ cli_nginx_dir  }}"
+    path: "{{ cli_nginx_dir }}"
     state: directory
 
-- include: docker_login.yml
+- set_fact:
+    cli_installation_mode="{{ openwhisk_cli.installation_mode }}"
```

Comment:
Why is the `set_fact` needed? Couldn't you just use `openwhisk_cli.installation_mode`everywhere?